subsys/net/lib/aws_iot/Kconfig

```diff
@@ -0,0 +1,75 @@
+#
+# Copyright (c) 2019 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+menuconfig AWS_IOT
+	bool "AWS IoT library"
+	select CLOUD_API
+	select MQTT_LIB
+	select MQTT_LIB_TLS
+	select AT_CMD
+
+if AWS_IOT
+
+config AWS_IOT_STATIC_IPV4
+	bool "Enable use of static IPv4"
+
+config AWS_IOT_STATIC_IPV4_ADDR
+	string "Static IPv4 address"
+	depends on AWS_IOT_STATIC_IPV4
+
+config AWS_IOT_SEC_TAG
+	int "Security tag to use for AWS IoT connection"
+	default 42
+
+config AWS_IOT_HOST_NAME
+	string "AWS IoT server hostname"
+	default "a34x44yyrk96tg-ats.iot.eu-central-1.amazonaws.com"
```

Comment:
Not sure we should provide a default value here, perhaps better to fail compile time with a sensible warning than attempt connection to a random hostname

Reply:
Agreed